spec/valkyrie/storage/fedora_spec.rb

```diff
@@ -49,4 +27,21 @@
 
     it_behaves_like "a Valkyrie::StorageAdapter"
   end
+
+  context 'no ldp gem' do
+    let(:error) { Gem::LoadError.new.tap { |err| err.name = 'ldp' } }
+    let(:error_message) do
+      "You are using the Fedora adapter without installing the ldp gem.  "\
+        "Add `gem 'ldp'` to your Gemfile."
```

Comment:
Extraneous detail.

Reply:
I don't think it's extraneous. This was reviewed before as part of another PR into this branch.